app/controllers/users_controller.rb

```diff
@@ -8,7 +8,7 @@ class UsersController < ApplicationController
   WHITELISTED_ATTRIBUTES = [:first_name, :last_name, :spire, :email,
                             :phone, :active, :reminders_enabled,
                             :change_notifications_enabled,
-                            { rosters: [], membership: [:admin] }].freeze
+                            { roster_ids: [], membership: [:admin] }].freeze
```

Comment:
I did this because it was annoying me and I kept getting confused about whether I had rosters or ids. Turns out, most of that confusion stemmed from `parse_roster_ids`, but I think this change is warranted either way.